src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,10 @@
+$circle-size: 40px;
+$circle-colore: #000;
+$millisecond-color: #f00;
+$second-arrow-color: #00f;
+$watch-size: 570px;
+$milisecond-arrow-duration: 1s;
+$second-arrow-duration: 60s;
+$arrow-width: 20px;
+$millisecond-lengh: 250px;
+$second-length: $millisecond-lengh / 1.5;
```

Comment:
Orphographic mistakes. I see that you write the same word in different ways (like "color and color", "length and length", "milisecond and millisecond"). Your future colleagues can have some problems with it.

Reply:
Agree) sometimes orphographic my bad side)